transport/src/main/java/io/servicefabric/transport/ConnectorHandshakeChannelHandler.java

```diff
@@ -20,12 +18,12 @@
 import java.util.concurrent.TimeoutException;
 
 /**
- * Duplex handler. On inbound recognizes only handshake message {@link TransportHandshakeData#Q_TRANSPORT_HANDSHAKE_SYNC_ACK}
- * (rest inbound messages unsupported and results in {@link TransportBrokenException}). On outbound may enqueue
- * messages. On 'channel active' starting handshake process.
+ * Duplex handler. On inbound recognizes only handshake message
+ * {@link TransportHandshakeData#Q_TRANSPORT_HANDSHAKE_SYNC_ACK} (rest inbound messages unsupported and results in
+ * {@link TransportBrokenException}). On outbound may enqueue messages. On 'channel active' starting handshake process.
  * <p/>
- * <b>NOTE:</b> this handler is not shareable (see {@link #sendMailbox}, {@link #handshakeTimeout}); and should always run
- * in different executor than io-thread.
+ * <b>NOTE:</b> this handler is not shareable (see {@link #sendMailbox}, {@link #handshakeTimeout}); and should always
+ * run in different executor than io-thread.
```

Comment:
For future: we just prove that switch between executors kills performance, so need to think again is it really need to run in different executor. 
@artem-v can you look?